Doc/requirements.txt

```diff
@@ -8,6 +8,8 @@ sphinx==3.2.1
 # version 3.2.1. It can be removed after bumping Sphinx version to at
 # least 3.5.4.
 docutils==0.17.1
+# Jinja version is pinned to a version compatible with Sphinx version 2.4.4.
```

Comment:
```suggestion
# Jinja version is pinned to a version compatible with Sphinx version 3.2.1.
```
The comment should be changed to match pinned Sphinx version.

Reply:
Yes, thanks!